plugins/modules/info.py

```diff
@@ -74,6 +75,37 @@
     choices: ['effective', 'user', 'default']
     default: 'effective'
     type: str
+  sort_dir:
```

Comment:
My two cents would be to group all the query parameters related to a particular resource under a top-level module argument. That way, we could ensure that the top-level argument list does not grow very large.
For e.g., for event channels

```
- query_parameters:
     - sort: name
     - limit: 10
     - dir: ASC
```

Reply:
Hi @anupamaloke 
I have addressed the review comments and updated the code accordingly. Please review the changes.